README.md

```diff
@@ -1368,11 +1440,67 @@ Send text to Pinecone's inference API to generate embeddings for documents and q
 	}
 	fmt.Printf("query embedding response: %+v", queryEmbeddingsResponse)
 
-	// << Send query to Pinecone to retrieve similar documents >>
+    // << Send query to Pinecone to retrieve similar documents >>
+```
+
+### Rerank documents
+
+Rerank documents in descending relevance-order against a query.
+
+**Note:** The `score` represents the absolute measure of relevance of a given query and passage pair. Normalized
+between [0, 1], the `score` represents how closely relevant a specific item and query are, with scores closer to 1
+indicating higher relevance.
+
+```go
+    ctx := context.Background()
+
+    pc, err := pinecone.NewClient(pinecone.NewClientParams{
+        ApiKey: "YOUR-API-KEY"
+	})
```

Comment:
nit: Indentation may be off here.

Reply:
It looks good on my screen; it's aligned with `pc`. What are you seeing?
<img width="610" alt="Screenshot 2024-10-21 at 4 48 22 PM" src="https://github.com/user-attachments/assets/24ac285a-e4f0-433a-b619-1c1d66000c2f">

Reply:
I think it's just gh formatting, nevermind.

Reply:
okay cool!